dev/python_version_test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Copyright 2020 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# USAGE: ./dev/python_version_test.py <python version> <env_name>
```

Comment:
nit: can you add this comment to the top of the other new test file?

Reply:
EDIT: if the other one is not meant to be run directly, maybe just add a comment there that says to use this file. If it is meant to be run directly as well, ignore the last sentence :)